app/ai-gateway/ai-audit-log-reference.md

```diff
@@ -40,116 +40,254 @@ You can also use [logging plugins](/plugins/?category=logging) to route these lo
 
 Each AI plugin returns a set of tokens. Log entries include the following details:
 
-<!--vale off-->
+
+### AI Proxy core logs
+
+The [AI Proxy](/plugins/ai-proxy/) and [AI Proxy Advanced](/plugins/ai-proxy/) plugins act as the main gateway for forwarding requests to AI providers. Logs here capture detailed information about the request and response payloads, token usage, model details, latency, and cost metrics. They provide a comprehensive view of each AI interaction.
+
 {% table %}
 columns:
   - title: Property
     key: property
   - title: Description
     key: description
 rows:
-  - property: "`ai.$PLUGIN_NAME.payload.request`"
-    description: The request payload.
-  - property: "`ai.$PLUGIN_NAME.payload.response`"
-    description: The response payload.
-  - property: "`ai.$PLUGIN_NAME.usage.prompt_token`"
+  - property: "`ai.proxy.payload.request`"
+    description: The request payload sent to the upstream AI provider.
+  - property: "`ai.proxy.payload.response`"
+    description: The response payload received from the upstream AI provider.
+  - property: "`ai.proxy.usage.prompt_token`"
     description: The number of tokens used for prompting.
-  - property: "`ai.$PLUGIN_NAME.usage.completion_token`"
+  - property: "`ai.proxy.usage.completion_token`"
     description: The number of tokens used for completion.
-  - property: "`ai.$PLUGIN_NAME.usage.total_tokens`"
-    description: The total number of tokens used.
-  - property: "`ai.$PLUGIN_NAME.usage.cost`"
-    description: The total cost of the request (input and output cost).
-  - property: "`ai.$PLUGIN_NAME.usage.time_per_token`"
-    description: |
-      {% new_in 3.8 %} The average time to generate an output token, in milliseconds.
-  - property: "`ai.$PLUGIN_NAME.meta.request_model`"
-    description:  The model used for the AI request.
-  - property: "`ai.$PLUGIN_NAME.meta.provider_name`"
-    description:  The name of the AI service provider.
-  - property: "`ai.$PLUGIN_NAME.meta.response_model`"
-    description:  The model used for the AI response.
-  - property: "`ai.$PLUGIN_NAME.meta.plugin_id`"
-    description:  The unique identifier of the plugin.
-  - property: "`ai.$PLUGIN_NAME.meta.llm_latency`"
+  - property: "`ai.proxy.usage.total_tokens`"
+    description: The total number of tokens used (input + output).
+  - property: "`ai.proxy.usage.input_tokens`"
+    description: The total number of input tokens (text + image + audio).
```

Comment:
input_tokens*, output_tokens* are all 3.11 new.

we might also want to mention that, text (chat, embeddings) responses uses prompt_token and completion_token, while
other non-text responses uses input_tokens and output_tokens